README.md

```diff
@@ -165,12 +165,22 @@ library
   default-language:    Haskell2010
 ```
 
-If you want to be able to import `Extra.*` modules when using `mixins` approach,
-you need to list those modules under `mixins` field as well, like this:
+This way, you can't access any other module of relude. If you want to use e.g.
```

Comment:
> you can't access any other module of relude

This sounds a bit misleading. I think that specifying that it's only about `Extra.*` modules could help to prevent misunderstanding.

Reply:
Hmm, I don't see how it is misleading, it is what is actually happening. `Relude.Unsafe` is also hidden, so it would also have to be mentioned.

Reply:
The thing is that we design the `Relude` module so that it reexports all other modules. So you do have access to the whole functional of all other modules except for `Extra.*` ones. Saying "you can't access any module" sounds very restrictive and can be interpreted as you don't have access to anything for those who don't know the internals of Relude.
I am not saying that your phrase is technically wrong. I am just saying that with the minimal effort we can prevent this sentence from being misunderstood or harder to get the initial point from.

Reply:
I think that leaving out this phrase (as it was before) can make this part easier to understand without missing any benefits.

Reply:
That's a good point, I will just remove the phrase!